### ISSUE TITLE
PHP 8.2: LDAP deprecation warnings in log

### ISSUE THREAD

**Issue by kwilczek** [roundcube/roundcubemail#9156]
Roundcube version: `1.6.3`.
PHP: `8.2`
OS: Debian "Bookworm" 12

We use LDAP in our setup and in the Roundcube logs we see a lot of this warrnings:

```
[30-Sep-2023 12:02:59 Europe/Warsaw] PHP Deprecated:  Creation of dynamic property Net_LDAP3_Result::$vlv is deprecated in /srv/roundcube/vendor/kolab/net_ldap3/lib/Net/LDAP3/Result.php on line 90
```

Those warnings probably have something to do with this:
https://wiki.php.net/rfc/deprecate_dynamic_properties

I also tried to suppress the warnings with `error_reporting` in `php.ini`, but without success (adding `~E_USER_DEPRECATED` to already enabled `~E_DEPRECATED`).
https://www.php.net/manual/en/errorfunc.constants.php

Kind regards,

**Comment by alecpl** [comment]
Fixed in https://git.kolab.org/rPNL39a754269fe6e5d94b0fe87215e89232876bda01.

**Comment by yolorkit** [comment]
> Fixed in https://git.kolab.org/rPNL39a754269fe6e5d94b0fe87215e89232876bda01.

Why wasn't this fix applied to 1.6.9?

**Comment by alecpl** [comment]
Because there was no new Net_LDAP3 release since.
